modules/installation-complete-user-infra.adoc

```diff
@@ -26,6 +29,24 @@ cluster on infrastructure that you provide.
 
 .Procedure
 
+. If instead of installing a new cluster you are upgrading an existing cluster from version 4.3 or from versions 4.4.0 to 4.4.11, run:
++
+----
+$ oc edit configs.samples cluster
+----
++
+Then, change the line
```

Comment:
I would consider restructuring this step to be part of the numbered procedure.

    . Change the line `managementState: Removed` to `managementState: Managed`.

Reply:
This step needs to remain under the conditional of "if you are not installing, but upgrading". If I promote it to being a separate step, it will escape that conditional.

Reply:
@rh-max Sounds good. Thanks!